routes/crypto/products.js

```diff
@@ -0,0 +1,73 @@
+const express = require('express')
+const router = express.Router()
+const authenticate = require('../../middlewares/authenticate')
+const productsController = require('../../controllers/crypto/productsController')
+
+// TODO: swager documantation
+/**
+ * @swagger
+ * /crypto/products:
+ *   post:
+ *     summary: Post product details to Database
+ *     tags:
+ *       - Crypto
+ *     parameters:
+ *       - in: body
+ *         data: productObject
+ *     responses:
+ *       201:
+ *         description: Product added successfully
+ *       409:
+ *         description: Product already exists
+ *       401:
+ *         description: Unauthorized
+ *         content:
+ *           application/json:
+ *             schema:
+ *               $ref: '#/components/schemas/errors/unAuthorized'
+ */
+router.post('/', authenticate, productsController.addNewProduct)
+
+/**
+ * @swagger
+ * /crypto/products:
+ *   get:
+ *     summary: Get all product from crypto site
+ *     tags:
+ *       - Crypto
+ *     responses:
+ *       200:
+ *         description: Product returned successfully
+ *       404:
+ *         description: Product not found
+ *       401:
+ *         description: Unauthorized
+ *         content:
+ *           application/json:
+ *             schema:
+ *               $ref: '#/components/schemas/errors/unAuthorized'
+ */
```

Comment:
Please considering adding 5xx errors to swagger docs. They are missing for most routes.
for e.g 
![image](https://user-images.githubusercontent.com/66248039/103657570-11f3b000-4f90-11eb-90c4-a77c3af70c4f.png)

Here, we need to write docs for 503 error. You can refer this: https://www.npmjs.com/package/boom#http-5xx-errors

Great work! 💯  @prakashchoudhary07 Thank you!

Reply:
Thank you, I will add it